packages/bindgen/src/templates/node-wrapper.ts

```diff
@@ -52,10 +52,10 @@ export function generate({ spec: rawSpec, file }: TemplateContext): void {
     };
   `);
   nodeLines.push(`
-    /*global global*/
+    /*global global, require */
     import { createRequire } from 'node:module';
-    const require = createRequire(import.meta.url);
-    const nativeModule = require("./realm.node");
+    const nodeRequire = typeof require === 'function' ? require : createRequire(import.meta.url);
```

Comment:
Unsure if this is being linted.
```suggestion
    const nodeRequire = typeof require === "function" ? require : createRequire(import.meta.url);
```

Reply:
Unfortunately, it's a limitation of the TS in TS code, that it isn't being linted 😞